frontend/src/Components/Map.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { tileNames, tilesMap } from "./tilesMap.js";
+import "./Map.css";
+
+class Map extends React.Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  randomGrass() {
+    let randomNum = Math.floor(Math.random() * 40);
+    if (randomNum === 6 || randomNum === 23) {
+      return "highGrass";
+    } else if (randomNum === 21) {
```

Comment:
When you encounter a `return`, the rest of the code is *not read*. You don't need the `else` afterwards, you can just chain plain `if`, it'll be easier to read

Reply:
Deleted "else if" replaced by "if"